packages/platform/example/lib/main.dart

```diff
@@ -0,0 +1,138 @@
+// ignore_for_file: public_member_api_docs
+import 'package:flutter/material.dart';
+import 'package:platform/platform.dart';
+
+void main() {
+  runApp(const MyApp());
+}
+
+class MyApp extends StatelessWidget {
+  const MyApp({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    const LocalPlatform platform = LocalPlatform();
+    return MaterialApp(
+      title: 'Platform Example',
+      home: Scaffold(
+        appBar: AppBar(
+          title: const Text('Platform Example'),
+        ),
+        body: Center(
+          child: SingleChildScrollView(
+            child: Column(
+              mainAxisAlignment: MainAxisAlignment.center,
+              children: <Widget>[
+                FormatDetails(
+                  title: 'Operating System:',
+                  value: platform.operatingSystem,
+                ),
+                FormatDetails(
+                  title: 'Number of Processors:',
+                  value: platform.numberOfProcessors.toString(),
+                ),
+                FormatDetails(
+                  title: 'Path Separator:',
+                  value: platform.pathSeparator,
+                ),
+                FormatDetails(
+                  title: 'Local Hostname:',
+                  value: platform.localHostname,
+                ),
+                FormatDetails(
+                  title: 'Environment:',
+                  value: platform.environment.toString(),
+                ),
+                FormatDetails(
+                  title: 'Executable:',
+                  value: platform.executable,
+                ),
+                FormatDetails(
+                  title: 'Resolved Executable:',
+                  value: platform.resolvedExecutable,
+                ),
+                FormatDetails(
+                  title: 'Script:',
+                  value: platform.script.toString(),
+                ),
+                FormatDetails(
+                  title: 'Executable Arguments:',
+                  value: platform.executableArguments.toString(),
+                ),
+                FormatDetails(
+                  title: 'Package Config:',
+                  value: platform.packageConfig.toString(),
+                ),
+                FormatDetails(
+                  title: 'Version:',
+                  value: platform.version,
+                ),
+                FormatDetails(
+                  title: 'Stdin Supports ANSI:',
+                  value: platform.stdinSupportsAnsi.toString(),
+                ),
+                FormatDetails(
+                  title: 'Stdout Supports ANSI:',
+                  value: platform.stdoutSupportsAnsi.toString(),
+                ),
+                FormatDetails(
+                  title: 'Locale Name:',
+                  value: platform.localeName,
+                ),
+                FormatDetails(
```

Comment:
The order of the `is*` values seems pretty random; please use alphabetical instead.